exercise_dirty.py

```diff
@@ -0,0 +1,30 @@
+foo = []
+
+from .models import Expense
+from collections import namedtuple
+
+MyExpense = namedtuple('F', ['type_', 'amount'])
+
+# test data
+foo.append(MyExpense('food', 4))
+foo.append(MyExpense('food', 3))
+foo.append(MyExpense('var', 3))
+foo.append(MyExpense('dog', 1))
+
+
+def summarizeExpenses(min_amount, input):
+    expenses = {}
+    for expense in input:
+        if expense.amount >= min_amount:
+            if not expense.type in expenses:
```

Comment:
I think this if/ else condition is more readable as:
if expense.type_ in expenses:
    expenses[expense.type_] = expenses[expense.type_] + expense.amount
else:
    expenses[expense.type_] = expense.amount